pcap-haiku.c

```diff
@@ -288,6 +299,37 @@ pcap_activate_haiku(pcap_t *handle)
 //	#pragma mark - pcap API
 
 
+static int
+can_be_bound(const char *name)
+{
+	if (strcmp(name, "loop"))
```

Comment:
`strcmp(name, "loop") != 0` might make it a bit clearer that this is testing for names that aren't "loop".